content/ops/security-ir.md

```diff
@@ -0,0 +1,180 @@
+# cloud.gov Incident Response process
+
+This document outlines cloud.gov's process for responding to security incidents. It outlines roles and responsibilities during and after incidents, and lays out the steps we'll take to resolve them.
+
+## Response process
+
+### Initiate
+
+An incident begins when someone (the *reporter*) becomes aware of a potential incident. We define "incident" broadly, following [NIST SP 800-61](http://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-61r2.pdf), as "a violation or imminent threat of violation of computer security policies, acceptable use policies, or standard security practices" (6). This is a deliberately broad definition, designed to encompass any scenario that might threaten the security of cloud.gov.
+
+The reporter should contact [FIXME: initial contact?] to become the response process. They should escalate [FIXME: how?] until they receive acknowledgment of their report.
+
+The first responder at this point becomes *Incident Commander* (IC), and carries out the next steps in the response. The IC's responsibility is coordination, not necessarily investigation. The IC's primary role is to guide the process.
+
+The first step for the IC is to create an issue in the [security-incidents](https://github.com/18f/security-incidents) github repo. Copy the following template to create the issue:
+
+```
+Short description of what's going on
+
+- **Status**: investigating
+- **Severity**: unknown
+- **Reporter**: who initially reported the issue
+- **IC**: Your Name
+- **Responders**: Any other responders
+
+Any extra details about the issue can go here.
+```
+
+The IC is responsible for keeping this issue up-to-date as investigation and remediation progresses. Everyone involved in the issue (*responders*) should leave notes as comments on the issue.
+
+#### Comms at the Initiate phase
+
+Note that at this point the issue's status is "investigating" - we haven't confirmed that it's really an issue yet. So, we should actually refer to this as just an "event" at this point; it doesn't become an "incident" until we've confirmed it. 
+
+At this phase, communications should follow these guidelines: 
+
+- Updates should be posted to the Github issue.
+- Real-time chat should happen in `#incident-response`
+- The IC may start a Google Hangout so that responders can share sensitive information not suitable for sharing in Github or Slack.
+- No mandatory reporting yet, though the IC may give relevant stakeholders an early notice if they deem in necessary.
+
+### Assess
+
+The next step is to assess the issue. We need to answer two questions:
+
+- Is this an incident (i.e., did the thing we suspect happen actually happen?)
+- If so, how severe is it? (This will determine how our response proceeds.)
+
+To answer these questions, the IC should form a response team. [FIXME: how to contact people here?]. The response team should work to confirm the issue and assess its impact. 
```

Comment:
Sure, but the one thing we need to avoid creating any delta or delay upon is GSA IR (and thus, the Office of the CISO set in policy) must be notified within 1 hour of even noticing something is *potentially* wrong. We gotta be all over that and not allow any local system policies (for cloud.gov or otherwise) to slow that down.

GSA IR is totally ok with that - they *want* unbaked or even false alarms, so they can clear the policy hurdles for us, so we focus on actually fixing the thing. cc @kimberbat